app/models/approval_request.rb

```diff
@@ -0,0 +1,8 @@
+class ApprovalRequest < ApplicationRecord
+  enum status: [:approve, :reject]
```

Comment:
nên để theo dạng key: value nhé, chỉ định giá trị cụ thể luôn 
```suggestion
  enum status: [approve: 0, reject: 1].freeze
```